components/TextFields/tests/snapshot/MDCOutlinedTextFieldSnapshotTests.m

```diff
@@ -0,0 +1,215 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialSnapshot.h"
+
+#import <UIKit/UIKit.h>
+
+#import "../../src/ContainedInputView/private/MDCTextControl.h"
+#import "MDCBaseTextFieldTestsSnapshotTestHelpers.h"
+#import "MDCTextControlSnapshotTestHelpers.h"
+#import "MaterialTextFields+ContainedInputView.h"
+
+@interface MDCOutlinedTextFieldTestsSnapshotTests : MDCSnapshotTestCase
```

Comment:
The corners on these outlines look incorrect to me. They have stretch artifacts. The corners are thicker than they should be.
Is this a known issue? part of how we are doing snapshot test? 

(im ok with a todo to acknowledge and fix in a follow up PR)

The existing outline textfield snapshots do not have them:
https://github.com/material-components/material-components-ios/blob/develop/snapshot_test_goldens/goldens_64/MDCTextFieldOutlinedControllerBaselineLeadingImageArabicSnapshotTests/testTextFieldWithShortHelperTextDisabled_11_2%402x.png

ps. I really wanted to add a comment to the snapshot file but couldn't figure out how.
https://github.com/material-components/material-components-ios/pull/8705/files?short_path=bd25f53#diff-bd25f53dedc310f907553486e4091cde

Reply:
See https://github.com/material-components/material-components-ios/pull/8705#discussion_r341711634